digitalpublicgoods/unido-industrial-analytics-platform.json

```diff
@@ -0,0 +1,57 @@
+{
+  "name": "UNIDO Industrial Analytics Platform",
+  "clearOwnership": {
+    "isOwnershipExplicit": "Yes",
+    "copyrightURL": "https://open.unido.org/projects/M0/projects/190326"
+  },
+  "platformIndependence": {
+    "mandatoryDepsCreateMoreRestrictions": "Yes",
+    "isSoftwarePltIndependent": "Yes",
+    "pltIndependenceDesc": "Open content"
+  },
+  "documentation": {
+    "isDocumentationAvailable": "Yes",
+    "documentationURL": [
+      "https://iap.unido.org/articles"
```

Comment:
Confirm whether the content articles can be accessed outside `IAP` in an open format. Thanks.